casa/Arrays/ArrayMath.h

```diff
@@ -904,7 +905,8 @@ T* expandRecursive (int axis, const IPosition& shp, const IPosition& mult,
                     const T* in, T* out, const IPosition& alternate);
 // Check array shapes for expandArray. It returns the alternate argument,
 // where possibly missing values are appended (as 0).
-IPosition checkExpandArray (IPosition& mult,
+// It fills in mult and inshp (with possibly missing axes of length 1) and outshp.
+IPosition checkExpandArray (IPosition& mult, IPosition& inshp,
```

Comment:
Could you explain (or document) the difference between `inshp` and `inShape`? The names don't speak for themselves.